mmv1/third_party/terraform/acctest/bootstrap_test_utils.go

```diff
@@ -361,19 +361,27 @@ func BootstrapSharedTestNetwork(t *testing.T, testId string) string {
 	return network.Name
 }
 
-type AddressSettings struct {
-	PrefixLength int
+type OptionalSettings struct {
```

Comment:
We should use different types per function! That'll stop us from accidentally using bad options, like AddressWithPrefixLength with BootstrapSharedServiceNetworkingConnection

Reply:
But we do need `AddressWithPrefixLength` with `BootstrapSharedServiceNetworkingConnection` for some cases, like https://github.com/GoogleCloudPlatform/magic-modules/blob/a007c7ce0f9498648bb61de3e9151b0639ed181b/mmv1/products/looker/Instance.yaml#L74C67-L74C67

Reply:
I think that's a coincidental overlap so far- merging these ones now will encourage merging all future examples too even if there isn't any overlap. With a settings object per bootstrap fn we'd define:

```
type AddressSettings struct {
	PrefixLength  int
}

type ServiceNetworkSettings struct {
	PrefixLength  int
	ParentService string
}
```